include/tudocomp/ds/uint_t.hpp

```diff
@@ -83,7 +83,7 @@ class uint_t: public IntegerBase<uint_t<bits>> {
     constexpr operator signed char() const { return m_data; }
 
     constexpr uint_t(char i): m_data(i) {}
-    constexpr uint_t& operator=(char data) { m_data = data; return *this; }
+    inline uint_t& operator=(char data) { m_data = data; return *this; }
     constexpr operator char() const { return m_data; }
```

Comment:
If I remember right this change happened as part of making clang happy. 

I think the version on master should work as-is though, since we have since merged our own fix for this issue. Could you remove the change and test it locally?